Magento/ruleset.xml

```diff
@@ -195,6 +195,10 @@
         <severity>7</severity>
         <type>warning</type>
     </rule>
+    <rule ref="Magento.Exceptions.LocalizedThrow">
+        <severity>7</severity>
```

Comment:
Please change the severity to `8`. This rule looks like:
> Magento specific code issue.

Reply:
Sorry for nit-picking, but could you please add rules in alphabetical order? So they will be ordered by severity then by name.